misc/roundtrip

```diff
@@ -126,7 +126,7 @@ IFS='
 
 tagfiles=$(find "$UNITS" -name expected.tags)
 for tags in $tagfiles; do
-    tagnames=$(sed -e 's/^\([^	]*\)	.*/\1/' "$tags" | limit_entries)
+    tagnames=$(cut -f1 < "$tags" | limit_entries)
```

Comment:
Doesn't the sed command line work on FreeBSD-11.4?
If it works, keep the commad line as is in this commit.
If you think using cut is better, you can make another commit for it (or another pull request).
As far as reading your commit log, you focus on t=... .

Reply:
I guess this change works with the bellow changes. Am I correct?